api/report_server.thrift

```diff
@@ -21,58 +21,55 @@ const string API_VERSION = '6.0'
 const i64 MAX_QUERY_SIZE = 500
 //=================================================
 
-//-----------------------------------------------------------------------------
-struct RunData{
+struct RunData {
   1: i64            runId,        // unique id of the run
   2: string         runDate,      // date of the run
   3: string         name,         // human readable name of the run
   4: i64            duration,     // duration of the run; -1 if not finished
   5: i64            resultCount,  // number of results in the run
   6: string         runCmd,       // the used check command
-  7: optional bool  can_delete    // true if codeCheckerDBAccess::removeRunResults()
+  7: optional bool  canDelete,    // true if codeCheckerDBAccess::removeRunResults()
                                   // is allowed on this run (see issue 151)
+  // this maps the detection status to its count
   8: map<shared.DetectionStatus, i32> detectionStatusCount
-                                  // this maps the detection status to its count
 }
 typedef list<RunData> RunDataList
 
-struct RunReportCount{
+struct RunReportCount {
   1: i64            runId,        // unique id of the run
   2: string         name,         // human readable name of the run
   3: i64            reportCount
 }
 typedef list<RunReportCount> RunReportCounts
 
-struct ReviewData{
+struct ReviewData {
   1: shared.ReviewStatus  status,
   2: string               comment,
   3: string               author,
   4: string               date
 }
 
-//-----------------------------------------------------------------------------
-struct ReportData{
-  1: string              checkerId,       // the qualified id of the checker that reported this
-  2: string              bugHash,         // This is unique id of the concrete report.
-  3: string              checkedFile,     // this is a filepath
-  4: string              checkerMsg,      // description of the bug report
-  5: i64                 reportId,        // id of the report in the current run in the db
-  6: i64                 fileId,          // unique id of the file the report refers to
-  7: i64                 line,            // line number or the reports main section (not part of the path)
-  8: i64                 column,          // column number of the report main section (not part of the path)
-  9: shared.Severity     severity         // checker severity
-  10: ReviewData          review           // bug review status informations.
+struct ReportData {
+  1: string                  checkerId,       // the qualified id of the checker that reported this
+  2: string                  bugHash,         // This is unique id of the concrete report.
+  3: string                  checkedFile,     // this is a filepath
```

Comment:
In case this is the `main file` which was given for the analyzer, we should use the same nomenclature.